app/src/main/kotlin/com/x8bit/bitwarden/ui/vault/feature/itemlisting/VaultItemListingViewModel.kt

```diff
@@ -914,10 +927,29 @@ class VaultItemListingViewModel @Inject constructor(
                     sendType = itemType.type.toSendItemType(),
                 )
             }
+
+            VaultItemListingState.DisplayItem.ItemType.DecryptionError -> {
+                showCipherDecryptionErrorItemClick(itemId = action.id)
+                return
+            }
         }
+
         sendEvent(event)
     }
 
+    private fun showCipherDecryptionErrorItemClick(itemId: String?) {
+        mutableStateFlow.update {
+            @Suppress("MaxLineLength")
+            it.copy(
+                dialogState = VaultItemListingState.DialogState.CipherDecryptionError(
+                    title = R.string.decryption_error.asText(),
+                    message = R.string.bitwarden_could_not_decrypt_this_vault_item_description_long.asText(),
+                    selectedCipherId = itemId,
```

Comment:
When the strings are moved to the `ui` module you can use `BitwardenString` instead of R.string.*` to reference them.